ui/app/templates/vault/cluster/access/reset-password.hbs

```diff
@@ -0,0 +1,3 @@
+<Page::UserpassResetPassword @backend={{this.model.backend}} @username={{this.model.username}} />
+
+{{outlet}}
```

Comment:
Do we need outlet here? It doesn't seem like there are any child routes rendering - but I have a tough time just reading ember templates/routing